forge-gui/res/cardsfolder/upcoming/evangel_of_synthesis.txt

```diff
@@ -0,0 +1,11 @@
+Name:Evangel of Synthesis
+ManaCost:U B
+Types:Creature Phyrexian Human Cleric
+PT:2/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw a card, then discard a card.
+SVar:TrigDraw:DB$ Draw | Cost$ T | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Mode$ TgtChoose | SubAbility$ DBPutCounter
```

Comment:
No SubAbility

Reply:
You still need to remove ` | SubAbility$ DBPutCounter`